models/instrument.py

```diff
@@ -979,8 +1061,9 @@ def get_gain_at_pixel( self, image, x, y, section_id=None ):
             return self.get_section( image.section_id ).gain
         elif section_id is not None:
             return self.get_section( section_id ).gain
-        elif sec.gain is not None:
-            return self.gain
+        # Can we remove this case? What is "sec" in this case??
```

Comment:
That must be vestigal code where merging didn't do the right thing.  I think sec was supposed to be the sensor section, but of course that's already hit in the previous elif.  I think these two lines can be deleted.